src/shellcode.rs

```diff
@@ -0,0 +1,1088 @@
+use std::mem;
+use winapi::shared::minwindef::MAX_PATH;
+
+static INJECTDLL_SC_32: &'static [u8] = &[
```

Comment:
```suggestion
static INJECTDLL_SC_32: &'static [u8; 0x2000] = &[
```
IMO slightly nicer to have this size explicitly confirmed by the compiler

Reply:
Done